### PR TITLE
Removing angular based dashboards

### PR DESCRIPTION
**Pull request by GregWhiteyBialas**

### CODE REVIEW

**Comment by GregWhiteyBialas** [comment]
This is last dashboard from this task: https://stackhpc.atlassian.net/browse/INFRA-629.

File: etc/kayobe/kolla/config/grafana/dashboards/openstack/redfish.json [hunk 1]
```diff
@@ -3290,541 +3578,286 @@
...
+  "schemaVersion": 39,
   "tags": [],
   "templating": {
     "list": [
       {
         "current": {
           "selected": false,
           "text": "Prometheus",
-          "value": "Prometheus"
+          "value": "PBFA97CFB590B2093"
```
Comment (dougszumski) [inline etc/kayobe/kolla/config/grafana/dashboards/openstack/redfish.json]: nit: revert

File: etc/kayobe/kolla/config/grafana/dashboards/openstack/redfish.json [hunk 2]
```diff
@@ -3841,10 +3874,10 @@
         "type": "datasource"
       },
       {
-        "allValue": null,
         "current": {
-          "text": "cpu-e-1041",
-          "value": "cpu-e-1041"
+          "selected": false,
+          "text": "hpc16",
```
Comment (dougszumski) [inline etc/kayobe/kolla/config/grafana/dashboards/openstack/redfish.json]: nit: Ideally we should strip these hostnames out - it should be able to handle re-populating them.

File: etc/kayobe/kolla/config/grafana/dashboards/openstack/redfish.json [hunk 3]
```diff
@@ -60,64 +57,37 @@
...
+      "pluginVersion": "10.4.2",
       "targets": [
         {
-          "expr": "count(up{job=\"redfish-exporter\"} == 1)",
+          "datasource": {
+            "type": "prometheus",
+            "uid": "${datasource}"
+          },
+          "editorMode": "code",
+          "expr": "count by(job) (redfish_up{env=\"hu-cloud\"})",
```
Comment (dougszumski) [inline etc/kayobe/kolla/config/grafana/dashboards/openstack/redfish.json]: Need to avoid site specific `hu-cloud`

**Review by dougszumski — CHANGES_REQUESTED** [review]
The main issue is a templating issue which prevents deployment:

```
TASK [grafana : Copying over extra configuration file] ****************************************************************************************
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ansible.errors.AnsibleUndefinedVariable: 'env' is undefined. 'env' is undefined
failed: [doug-aio] (item={'path': '/home/ubuntu/src/kayobe-config/etc/kolla/config/grafana/redfish.json', 'mode': '0640', 'isdir': False, 'ischr': False, 'isblk': False, 'isreg': True, 'isfifo': False, 'islnk': False, 'issock': False, 'uid': 1000, 'gid': 1003, 'size': 92743, 'inode': 527227, 'dev': 64513, 'nlink': 1, 'atime': 1720609092.533431, 'mtime': 1720609092.1894255, 'ctime': 1720609092.533431, 'gr_name': 'kolla', 'pw_name': 'ubuntu', 'wusr': True, 'rusr': True, 'xusr': False, 'wgrp': False, 'rgrp': True, 'xgrp': False, 'woth': False, 'roth': False, 'xoth': False, 'isuid': False, 'isgid': False}) => {"ansible_loop_var": "item", "changed": false, "item": {"atime": 1720609092.533431, "ctime": 1720609092.533431, "dev": 64513, "gid": 1003, "gr_name": "kolla", "inode": 527227, "isblk": false, "ischr": false, "isdir": false, "isfifo": false, "isgid": false, "islnk": false, "isreg": true, "issock": false, "isuid": false, "mode": "0640", "mtime": 1720609092.1894255, "nlink": 1, "path": "/home/ubuntu/src/kayobe-config/etc/kolla/config/grafana/redfish.json", "pw_name": "ubuntu", "rgrp": true, "roth": false, "rusr": true, "size": 92743, "uid": 1000, "wgrp": false, "woth": false, "wusr": true, "xgrp": false, "xoth": false, "xusr": false}, "msg": "AnsibleUndefinedVariable: 'env' is undefined. 'env' is undefined"}`
```

Please can you check it deploys ok and update the PR?

File: etc/kayobe/kolla/config/grafana/dashboards/openstack/redfish.json (hunk 3 above)
Comment (GregWhiteyBialas) [reply etc/kayobe/kolla/config/grafana/dashboards/openstack/redfish.json]: Fixed.

File: etc/kayobe/kolla/config/grafana/dashboards/openstack/redfish.json (hunk 2 above)
Comment (GregWhiteyBialas) [reply etc/kayobe/kolla/config/grafana/dashboards/openstack/redfish.json]: Fixed. It appears that Grafa, during export also keeps "selected" values in dashboards. Putting empty values makes grafana using default, which is in this case, first hostname in the list.

File: etc/kayobe/kolla/config/grafana/dashboards/openstack/redfish.json (hunk 1 above)
Comment (GregWhiteyBialas) [reply etc/kayobe/kolla/config/grafana/dashboards/openstack/redfish.json]: Fixed. It appears that Grafa, during export also keeps "selected" values in dashboards. Putting empty values makes grafana using default value.

Edit: wording.

**Comment by GregWhiteyBialas** [comment]
@dougszumski I can't reproduce problem with deployment and missing "env" variable. I have deployed it in ci-aio env without problems. Can check again?

**Comment by dougszumski** [comment]
> @dougszumski I can't reproduce problem with deployment and missing "env" variable. I have deployed it in ci-aio env without problems. Can check again?

Thanks, my bad. I had a copy lurking in the config/grafana folder which triggered the templating attempt

**Comment by dougszumski** [comment]
I currently don't have an environment to test this in outside of an aio. It looks fine in the aio, and I don't see any warnings about Angular but obviously I don't have any data. We can follow up with any fixes if required - it's a good step forward, thanks @GregWhiteyBialas .

**Review by dougszumski — DISMISSED** [review]

File: etc/kayobe/kolla/config/grafana/dashboards/openstack/redfish.json [hunk 4]
```diff
@@ -78,7 +78,11 @@
             "uid": "${datasource}"
           },
           "editorMode": "code",
+<<<<<<< HEAD
```
Comment (dougszumski) [inline etc/kayobe/kolla/config/grafana/dashboards/openstack/redfish.json]: Needs removing

**Review by dougszumski — CHANGES_REQUESTED** [review]

**Review by dougszumski — APPROVED** [review]
Thanks @GregWhiteyBialas , all warnings resolved. I don't have access to a test env currently, but will try on sms in the near future.

Please squash the commits before merging

**Comment by dougszumski** [comment]
> Thanks @GregWhiteyBialas , all warnings resolved. I don't have access to a test env currently, but will try on sms in the near future.
> 
> Please squash the commits before merging

Argh, auto-merge caught me out. Not to worry.

**Pull request merged**